pkg/client/cache/listwatch_test.go

```diff
@@ -117,7 +117,7 @@ func TestListWatchesCanWatch(t *testing.T) {
 		{
 			location: buildLocation(
 				testapi.Default.ResourcePathWithPrefix("watch", "nodes", api.NamespaceAll, ""),
-				buildQueryValues(url.Values{"resourceVersion": []string{""}})),
+				buildQueryValues(url.Values{})),
```

Comment:
Typo in your second comment? The two parties are the same.

And as we are on it, can we get rid of the buildQueryValues() function? I think it's only useful when we have v1beta1 and v1beta2.